pycheribuild/projects/cross/cheribsd.py

```diff
@@ -1242,7 +1242,7 @@ def _remove_schg_flag(self, *paths: "typing.Iterable[str]"):
 
     def _remove_old_rootfs(self):
         if not self.config.skip_buildworld:
-            if self.install_as_root:
+            if self.install_as_root and not self.crossbuild:
```

Comment:
I'd prefer if this was done using a `if OSInfo.IS_FREEBSD` or `shutil.which("chflags")` check inside `_remove_schg_flag()`.